packages/atomic/cypress/e2e/facets/automatic-facet-builder/automatic-facet-builder.cypress.ts

```diff
@@ -0,0 +1,70 @@
+import {TestFixture} from '../../../fixtures/test-fixture';
+import {
+  assertConsoleError,
+  assertContainsComponentError,
+} from '../../common-assertions';
+import {addAutomaticFacetBuilder} from './automatic-facet-builder-actions';
+import {
+  assertCollapseAutomaticFacets,
+  assertContainsAutomaticFacet,
+  automaticFacetBuilderComponent,
+} from './automatic-facet-builder-assertions';
+
+describe('Automatic Facet Builder Test Suites', () => {
+  it('should throw an error when desiredCount property is invalid', () => {
+    new TestFixture()
+      .with(addAutomaticFacetBuilder({'desired-count': 'potato'}))
+      .init();
+    assertConsoleError();
+    assertContainsComponentError(
+      {
+        shadow: () => cy.get(automaticFacetBuilderComponent).shadow(),
+      },
+      true
+    );
+  });
+
+  it('should display atomic-automatic-facet when desiredCount is valid', () => {
+    new TestFixture()
+      .with(addAutomaticFacetBuilder({'desired-count': '1'}))
+      .init();
+    assertContainsAutomaticFacet();
+  });
+
+  it('should throw an error when areCollapsed property is invalid', () => {
+    new TestFixture()
+      .with(addAutomaticFacetBuilder({'are-collapsed': 'potato'}))
+      .init();
+    assertConsoleError();
+    assertContainsComponentError(
+      {
+        shadow: () => cy.get(automaticFacetBuilderComponent).shadow(),
+      },
+      true
+    );
+  });
+
+  it('should collapse the facets when areCollapsed property is `true`', () => {
```

Comment:
It's up for debate as to whether this is a worthwhile test, but seeing as the use-case for this prop is to collapse _**all automatic facets**_, I feel like it would be appropriate to use a `desired-count` that is greater than one. This way, we can confirm that it doesn't just collapse the top automatic facet, but all automatic facets.

This might be splitting hairs though, so I'm curious to hear others thoughts on this 🙂